### PR TITLE
Add Estonian Open-EID 3.12.4 as eid-ee

### PR DESCRIPTION
**Pull request by jaantoots**
# Electronic identity card software for Estonia

Installs the system and web browser components for authentication and digital signing.

Additional software is available from the Mac App Store (and usually installed); the package installer opens the App Store automatically and I have also included a link in the Caveats.

(Cask is named following the instructions in the homebrew-eid repository, rather than the token reference.)

### Checklist

- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:

- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-eid/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-eid/issues) where that cask was already refused.
- [ ] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/eid-ee.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  license :gpl
+
+  pkg 'Open-EID.pkg'
+
+  uninstall script: {
+                      executable: 'uninstall.sh',
+                      input:      %w[y],
+                    }
+
+  caveats "DigiDoc3 Client and ID-card Utility are available in the App Store:\n"\
```
Comment (vitorgalvao) [inline Casks/eid-ee.rb]: Why does this matter?

Comment (vitorgalvao) [inline Casks/eid-ee.rb]: Also, please use the [`undent` method](https://github.com/caskroom/homebrew-cask/blob/fe4fd74a9eb5bd5f8c046174fd943e5099123e8f/Casks/veracrypt.rb#L17).

Comment (jaantoots) [inline Casks/eid-ee.rb]: Updated to use `undent`.

Most users would most probably like to install the linked apps as well. The DigiDoc3 Client is required for opening, verifying and signing documents. The ID-card Utility for card info and changing PIN codes, etc.

Also, the package installer opens the [App Store page](http://appstore.com/mac/ria) anyway, so I think it also serves as an explanation. The official [homepage](https://installer.id.ee/?lang=eng) lists the installer included in this Cask and those apps side by side, which seems like an implicit suggestion that you should probably get both.

File: Casks/eid-ee.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  pkg 'Open-EID.pkg'
+
+  uninstall script: {
+                      executable: 'uninstall.sh',
+                      input:      %w[y],
+                    }
+
+  caveats <<-EOS.undent
+    DigiDoc3 Client and ID-card Utility are available in the App Store:
+    http://appstore.com/mac/ria
```
Comment (vitorgalvao) [inline Casks/eid-ee.rb]: Indent this line two spaces further.

File: Casks/eid-ee.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+cask 'eid-ee' do
+  version '3.12.4'
+  sha256 'a5a7b2824da4efb9479e4b70a952981d41c50a75b00839d0216133b04e32b5b5'
+
+  url 'https://installer.id.ee/media/osx/Open-EID_3.12.4.1666.dmg'
```
Comment (vitorgalvao) [inline Casks/eid-ee.rb]: `url "https://installer.id.ee/media/osx/Open-EID_#{version}.dmg"`

File: Casks/eid-ee.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+cask 'eid-ee' do
+  version '3.12.4'
```
Comment (vitorgalvao) [inline Casks/eid-ee.rb]: `version '3.12.4.1666'`

**Comment by jaantoots** [comment]
Made the requested changes.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**